CONTRIBUTING.md

```diff
@@ -1,11 +1,8 @@
-Contributing to Fluent Assertions
------------
+# Contributing to Fluent Assertions
 
-No open-source projection is going to be successful without contributions. After we decided to move to Github, the involvement of the .NET community has increased significantly. However, contributing to this project involves a few steps that will seriously increase the chance we will accept it.
+No open-source project is going to be successful without contributions. After we decided to move to Github, the involvement of the .NET community has increased significantly. However, contributing to this project involves a few steps that will seriously increase the chance we will accept it.
 
-* The [Pull Request](https://help.github.com/articles/using-pull-requests) is targeted at the `develop` branch.
+* The [Pull Request](https://help.github.com/articles/using-pull-requests) is targeted at the `master` branch.
 * The code complies with the [Coding Guidelines for C# 3.0, 4.0 and 5.0](http://csharpguidelines.codeplex.com/)/.
-* The changes are covered by a new or existing set of unit tests which follow the Arrange-Act-Assert syntax such as is used [in this example](https://github.com/fluentassertions/fluentassertions/blob/daaf35b9b59b622c96d0c034e8972a020b2bee55/Tests/FluentAssertions.Shared.Specs/BasicEquivalencySpecs.cs#L33). 
+* The changes are covered by a new or existing set of unit tests which follow the Arrange-Act-Assert syntax such as is used [in this example](https://github.com/fluentassertions/fluentassertions/blob/daaf35b9b59b622c96d0c034e8972a020b2bee55/Tests/FluentAssertions.Shared.Specs/BasicEquivalencySpecs.cs#L33).
 * If the contribution affects the documentation, please update [**documentation.md**](https://github.com/fluentassertions/fluentassertions/blob/master/docs/documentation.md), which is published on the [website](http://fluentassertions.com/documentation.html).
```

Comment:
The documentation link is also broken. It should be https://fluentassertions.com/documentation/

Reply:
Sure!

I also replaced the link to the `documentation.md` with a direct link to the docs folder in the repo.